examples/QmlBarcodeReader/cmake/Locations.cmake

```diff
@@ -0,0 +1,37 @@
+set(COMPANY "Scythe Studio")
+set(COPYRIGHT "Copyright (c) 2022 Scythe Studio. All rights reserved.")
```

Comment:
This should be updated for the preamble from the open-source license used in project